lib/Models/Terria.ts

```diff
@@ -876,16 +876,20 @@ export default class Terria {
     const configParams =
       aspects["terria-config"] && aspects["terria-config"].parameters;
 
-    configParams.initializationUrls =
-      aspects["terria-config"] && aspects["terria-config"].initializationUrls;
+    // TODO: what purpose does this serve? it's not a config parameter
+    // configParams.initializationUrls =
+    //   aspects["terria-config"] && aspects["terria-config"].initializationUrls;
```

Comment:
Agreed, I think this can be culled, from my reading of the code it serves no purpose.

Reply:
![image](https://user-images.githubusercontent.com/9134432/95289854-8a76a200-08b7-11eb-994d-47585cadea56.png)

Thanks for confirming 😀

Reply:
I gitblame you @soyarsauce 😂 
![Screen_2020_10_07](https://user-images.githubusercontent.com/6735870/95292167-b3e5fc80-08bc-11eb-9d24-310ee7738d6f.jpg)

Reply:
Hahaha sorry for misleading with half-pseudo-code!! You'll note my suggestion doesn't change the `initializationUrls` key on `configParams`. It looks like a naming problem, should have been called `initUrls` or something - `const initUrls = aspects["terria-config"] && aspects["terria-config"].initializationUrls;` and done what you will with the list of init urls there.

Anyway the extra mocked config map tests will help somewhat ensure things are loaded the right way 👍 it was quite difficult to verify without those unit tests.